cmd/bosun/web/web.go

```diff
@@ -119,6 +124,25 @@ func Listen(listenAddr string, devMode bool, tsdbHost string) error {
 	router.Handle("/api/tagv/{tagk}/{metric}", JSON(TagValuesByMetricTagKey))
 	router.Handle("/api/tagsets/{metric}", JSON(FilteredTagsetsByMetric))
 	router.Handle("/api/opentsdb/version", JSON(OpenTSDBVersion))
+	router.Handle("/api/annotate", JSON(AnnotateEnabled))
+
+	// Annotations
+	if schedule.Conf.AnnotateEnabled() {
+		var err error
+		index := schedule.Conf.AnnotateIndex
+		if index == "" {
+			index = "annotate"
+		}
+		annotateBackend, err = backend.NewElastic(schedule.Conf.AnnotateElasticHosts, index)
+		if err != nil {
+			return err
+		}
+		if err := annotateBackend.InitBackend(); err != nil {
+			return err
+		}
+		web.AddRoutes(router, "/api", []backend.Backend{annotateBackend}, false, false)
```

Comment:
was thinking it might be nice if your ui has direct query/filter/ raw viewing support.